src/RadioButton/RadioButton.js

```diff
@@ -232,8 +229,7 @@ const RadioButton = React.createClass({
         iconStyle={mergedIconStyle}
         labelStyle={mergedLabelStyle}
         labelPosition={labelPosition}
-        onSwitch={this._handleCheck}
-        onParentShouldUpdate={this._handleStateChange}
```

Comment:
I have two options:
1. Change `onParentShouldUpdate` to not be mandatory
2. Use something like:
```js
const noop = () => {};
...
onParentShouldUpdate={noop}
```